buddyboss/buddyboss-platform
# Fixed JS Error while deleting repeater field

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #845 

### How to test the changes in this Pull Request:

1. Go to BuddyBoss > Profile > Create Repeater field set
2. Add field like Paragraph Text
3. Go to User Profile and fields and list repeater field
4. Delete the repeater field. Check console. There is no error now.

### Proof Screenshots or Video
https://www.loom.com/share/4d00c2a5138942a88dae8507ca330ec1



### Other information:




### Changelog entry

> Fixed JS Error while deleting repeater field
